.releaserc.yml

```diff
@@ -0,0 +1,44 @@
+branches:
+  - master
+plugins:
+  - - "@semantic-release/commit-analyzer"
+    - releaseRules:
+        - breaking: true
+          release: major
+        - revert: true
+          release: patch
+        - type: feat
+          release: minor
+        - type: fix
+          release: patch
+        - type: perf
+          release: patch
+        - type: docs
+          release: patch
+  - - "@semantic-release/release-notes-generator"
+    - preset: conventionalcommits
+      presetConfig:
+        types:
+          - type: build
+            section: Build system / dependencies
+          - type: ci
+            section: CI
+          - type: docs
+            section: Documentation
+          - type: feat
+            section: Features
+          - type: fix
+            section: Bug fixes
+          - type: perf
+            section: Performance
+          - type: refactor
+            section: Refactoring
+          - type: text
+            section: Testing
+  - "@semantic-release/changelog"
+  - "@semantic-release/github"
+  - "@semantic-release/npm"
+  - - "@semantic-release/git"
+  - assets:
+      - CHANGELOG.md
+      - package.json
```

Comment:
That might be best to not have any assets.
The changelog will be published with the GitHub Release, what do you think? :smile:

Reply:
# `CHANGELOG.md`
This file already exists in this repo, so I might as well setup semantic-release to update it. 😃
Using a changelog file is also a widely accepted and implemented convention, seen in many major projects, including standard/standard.

# `package.json`
@semantic-release/npm updates the package.json version, so might as well commit that.